### ISSUE TITLE
no-page-pause does not produce warning when the object variable name is other than "page"

### ISSUE THREAD

**Issue by gajus** [playwright-community/eslint-plugin-playwright#124]
Produces warning:

```ts
await page.pause();
```

Does not produce warning:

```ts
await indyPage.pause();
```

I would suggest that the rule is extended to catch when `pause()` is used with `page` or any variable that ends with `Page`.

**Comment by mskelton** [comment]
My only concern with that is the potential for false positives. Let me think more about it

**Comment by mskelton** [comment]
Rolling this and some other related issues into https://github.com/playwright-community/eslint-plugin-playwright/issues/131
